### PR TITLE
RavenDB-17107 Sharding - Studio work

### PR DESCRIPTION
**Pull request by ml054**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17107

### Additional description

Base sharding UI work 

### Type of change

- New feature

### How risky is the change?

- Low 

### Backward compatibility

- Not relevant

### UI work

- It requires further work in the Studio - this is only WIP

### CODE REVIEW

**Comment by ayende** [comment]
Can you add the screen shots?

**Comment by ml054** [comment]
image

image


Doesn't look impressive but I'd like to allow to choose shard or shard group.

**Comment by ayende** [comment]
Thanks

File: src/Raven.Server/Documents/ShardedHandlers/ShardedChangesHandler.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using Raven.Server.Documents.Sharding;
+using Raven.Server.Routing;
+
+namespace Raven.Server.Documents.ShardedHandlers
+{
+    public class ShardedChangesHandler : ShardedRequestHandler
+    {
+        [RavenShardedAction("/databases/*/changes", "GET")]
+        public async Task GetChanges()
+        {
```
Comment (ppekrol) [inline src/Raven.Server/Documents/ShardedHandlers/ShardedChangesHandler.cs:18]: DevelopmentHelper.ToDo

File: src/Raven.Server/Documents/ShardedHandlers/ShardedDatabaseNotificationCenterHandler.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+using Raven.Server.Documents.Sharding;
+using Raven.Server.Routing;
+
+namespace Raven.Server.Documents.ShardedHandlers
+{
+    public class ShardedDatabaseNotificationCenterHandler : ShardedRequestHandler
+    {
+        [RavenShardedAction("/databases/*/notification-center/watch", "GET")]
+        public async Task Get()
+        {
```
Comment (ppekrol) [inline src/Raven.Server/Documents/ShardedHandlers/ShardedDatabaseNotificationCenterHandler.cs:12]: DevelopmentHelper.ToDo

**Pull request merged**